collectors/gsuite/README.md

```diff
@@ -10,35 +10,61 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 # Installation
 
-Refer to [CF template readme](./cfn/README.md) for installation instructions.
+### 1. G Suite Setup
+
+1. Create a Custom admin role. [Link](https://support.google.com/a/answer/2406043)
+2. Create a custom admin role with the 'Reports' privilege assigned
+3. Create an admin user, assigning the previously created role
+4. Enable API access for G Suite. [Link](https://support.google.com/a/answer/60757?authuser=3)
+
+### 2. Google Cloud platform Setup
+
+1. how to create a Google Cloud project [Link](https://console.cloud.google.com/home/dashboard)<br />
```

Comment:
Change this from `how to create a Google Cloud project` to `Create a Google Cloud project`